tests/nnapi/specs/V1_1/transpose_dynamic.mod.py

```diff
@@ -0,0 +1,23 @@
+import dynamic_tensor
+model = Model()
+model_input_shape = [1, 2, 2, 1]
+
+dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input_shape)
+test_node_input = dynamic_layer.getTestNodeInput()
+
+perms = Parameter("perms", "TENSOR_INT32", "{4}", [0,2,1,3])
+
+# write Transpose test. input is `test_node_input`
+# note output shape is dynamic and not known.
+# However, we need to provide any shape that can hold test output. Otherwise, TestGenerated.cpp will fail
+model_output = Output("output", "TENSOR_FLOAT32", "{1,4}")
```

Comment:
Could you use the expected shape?
There has been some changes: https://github.com/Samsung/ONE/pull/1380/files#diff-e77c1ab7a3495d9b6dcc45b32525022eR51
Also could you change the comment from 10~12?

Reply:
Thanks for spotting this. will address the change.